invenio_app_ils/documents/mappings/os-v1/documents/document-v1.0.0.json

```diff
@@ -2,24 +2,48 @@
   "settings": {
```

Comment:
Currently updated in document-v1.0.0 for better review of the changes
Move to v2.0.0 when finalised